components/Projects/Projects.js

```diff
@@ -0,0 +1,29 @@
+class Projects {
+  render() {
+    let htmlCatalog = "";
+    PROJECTS.forEach(({ id, imageUrl, title, text }) => {
```

Comment:
Why spacing is different in this file? Code style should be united across all files.